docs/psoc6/quickref.rst

```diff
@@ -633,19 +633,31 @@ Constructor
 Methods
 -------
```

Comment:
The headlines need to have 1 level lower (Methods, Constructor, etc.)

Reply:
And to keep consistency with the rest of the classes, I would name this "PDM-PDM bus"

Reply:
Noted